src/extensions/default/CSSCodeHints/styles/brackets-css-hints.css

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2014 Adobe Systems Incorporated. All rights reserved.
+ * Copyright (c) 2015 Adobe Systems Incorporated. All rights reserved.
```

Comment:
We never update the copyright date on files. This is the date the file was added to code base.